File: PArticulo/PArticulo/Articulo.cs
using System;

namespace PArticulo {

	public class Articulo {

		public Articulo (long id, string nombre, decimal? precio, long? categoria) {
			Id = id;
			Nombre = nombre;
			Precio = precio;
			Categoria = categoria;
		}

		public long Id 		   { set; get; }
		public string Nombre   { set; get; }
		public decimal? Precio { set; get; }
		public long? Categoria { set; get; }
	}
}

File: HMySql/src/main/java/org/institutpserpis/ad/Articulo.java
package org.institutpserpis.ad;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

@Entity
public class Articulo {
	private long id;
	private String nombre;
	private Double precio;
	private long categoria;
	
	@Id
	@GeneratedValue(strategy=GenerationType.IDENTITY)
	public long getId() {
		return id;
	}
	public void setId(long id) {
		this.id = id;
	}
	public String getNombre() {
		return nombre;
	}
	public void setNombre(String nombre) {
		this.nombre = nombre;
	}
	public Double getPrecio() {
		return precio;
	}
	public void setPrecio(Double precio) {
		this.precio = precio;
	}
	public long getCategoria() {
		return categoria;
	}
	public void setCategoria(long categoria) {
		this.categoria = categoria;
	}
	
}
